packages/cli/src/cli/commands/start.ts

```diff
@@ -0,0 +1,54 @@
+import { spawn } from "node:child_process";
+import { build, createApibara, prepare, writeTypes } from "apibara/core";
+import { runtimeDir } from "apibara/runtime/meta";
+import { defineCommand } from "citty";
+import { join, resolve } from "pathe";
+import { commonArgs } from "../common";
+
+export default defineCommand({
+  meta: {
+    name: "start",
+    description: "Start one indexer",
+  },
+  args: {
+    ...commonArgs,
+    indexer: {
+      type: "string",
+      description: "The indexer to start",
+      required: true,
+    },
+    preset: {
+      type: "string",
+      description: "The preset to use",
+    },
+  },
+  async run({ args }) {
+    const rootDir = resolve((args.dir || args._dir || ".") as string);
+    const apibara = await createApibara({
+      rootDir,
+    });
+
+    apibara.logger.start("Building");
+
+    apibara.options.entry = join(runtimeDir, "start.mjs");
+
+    await prepare(apibara);
+    await writeTypes(apibara);
+    await build(apibara);
```

Comment:
IMHO when user runs `apibara start` command, it should just run the build files generated from `apibara build` and not build it again and run it.

Reply:
Yeah I was unsure about this too. I think you're right and we should print an error message if the user hasn't built the app first (like Next.js does, message included for reference).

```
   ▲ Next.js 15.0.2
   - Local:        http://localhost:3000

 ✓ Starting...
Error: Could not find a production build in the '.next' directory. Try building your app with 'next build' before starting the production server. https://nextjs.org/docs/messages/production-start-no-build-id
    at setupFsCheck (/home/fra/.local/share/pnpm/store/v3/tmp/dlx-122690/node_modules/.pnpm/next@15.0.2_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/server/lib/router-utils/filesystem.js:149:19)
    at async initialize (/home/fra/.local/share/pnpm/store/v3/tmp/dlx-122690/node_modules/.pnpm/next@15.0.2_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/server/lib/router-server.js:65:23)
    at async Server.<anonymous> (/home/fra/.local/share/pnpm/store/v3/tmp/dlx-122690/node_modules/.pnpm/next@15.0.2_react-dom@18.3.1_react@18.3.1/node_modules/next/dist/server/lib/start-server.js:266:36)
```

Reply:
BTW love that Next.js has an "errors" section in their docs. We should do that too.